src/components/Callout/Callout.tsx

```diff
@@ -3,12 +3,21 @@ import { Message, View } from '@aws-amplify/ui-react';
 interface CalloutProps {
   info?: boolean;
   warning?: boolean;
+  backgroundColor?: string;
```

Comment:
Instead of changing how the callout component is currently being used I've exposed the backgroundColor property of the underlying Message component.  If both backgroundColor and the warning flag are used, the backgroundColor property will win out for backgroundColor styling